packages/react-components/src/components/VideoGallery/styles/FloatingLocalVideo.styles.ts

```diff
@@ -34,8 +34,9 @@ export const SMALL_FLOATING_MODAL_SIZE_PX = { width: 64, height: 88 };
 
 /**
  * Large floating modal width and height in rem for large screen
+ * Aspect ratio: 16:9
  */
-export const LARGE_FLOATING_MODAL_SIZE_PX = { width: 160, height: 120 };
+export const LARGE_FLOATING_MODAL_SIZE_PX = { width: 215, height: 120 };
```

Comment:
I like this as a quick fix that covers most use cases, we should definitely keep this change and push this PR in. That being said do we have a plan to also address non-16:9 ratios? (e.g. portrait cameras such as mobiles)

Reply:
(If this just works (everything else plays nicely with us adjusting this width), we could use the incoming stream size we get from beta calling sdk to set the width of the local preview dynamically)

Reply:
For Mobile, it uses the width and height from line 33
`export const SMALL_FLOATING_MODAL_SIZE_PX = { width: 64, height: 88 };`.

It's not quite the usual 4:3 aspect ratio that cameras use... I tested around with different width and heights for mobile but unfortunately, I am unable to get the full camera view. either the height, width, or both get cropped (zoomed in despite being the correct aspect ratio).

So for now, this fix is primarily for Desktop.

Reply:
I'll continue to investigate why mobile zooms in despite changing the modal to the correct aspect ratio. 
But, as of right now, the current modal size for mobile local video tiles is relatively accurate. Crops very slightly on left and right.

Reply:
> (If this just works (everything else plays nicely with us adjusting this width), we could use the incoming stream size we get from beta calling sdk to set the width of the local preview dynamically)

Plays well on desktop! Not so well for mobile from my manual aspect ratio testing.